### PR TITLE
Setup on off st

### PR DESCRIPTION
**Pull request by SammyT13**

### CODE REVIEW

File: lightbulb-challenge-sammy/src/App.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      <h1>Lightbulb Challenge</h1>
+      <h2>Turn the lights on or off, by clicking any box.</h2>
+      <LightSwitch />
+      <LightSwitch />
+      <LightSwitch />
+    </>
+  )
+}
+
+export default App
```
Comment (chrisyhlee) [inline lightbulb-challenge-sammy/src/App.js:18]: Clean App.js page! Great work

File: lightbulb-challenge-sammy/src/components/LightSwitch.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    // Function that updates State Values (sets it value to either true / false)
+    const lightsOnOff = () => {
+        setLightSwitchStatus(lightSwitchStatus === false ? true : false)  
+    }
+    
+    // if light switch is true = 'On', else 'Off'
+    let switchOnOff = lightSwitchStatus === true ? 'On' : 'Off'
+    // if light switch is true = 'yellow', else 'white'
+    let bulbOnOff = lightSwitchStatus === true ? 'yellow' : 'white'
```
Comment (chrisyhlee) [inline lightbulb-challenge-sammy/src/components/LightSwitch.js:17]: Nice use of the ternary operators for the switch status and background color! And very clean again!

File: lightbulb-challenge-sammy/src/App.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import LightSwitch from './components/LightSwitch';
+import './App.css';
+
+// Display component only
+const App = () => {
+  return (
+    <>
+      <h1>Lightbulb Challenge</h1>
+      <h2>Turn the lights on or off, by clicking any box.</h2>
```
Comment (jamesquillen) [inline lightbulb-challenge-sammy/src/App.js:10]: Good instruction to the user.

File: lightbulb-challenge-sammy/src/components/LightSwitch.js (hunk 2 above)
Comment (jamesquillen) [inline lightbulb-challenge-sammy/src/components/LightSwitch.js:17]: Good comments and use of the ternary operator.

**Review by jamesquillen — COMMENTED** [review]
Great work. Very clean code with good comments.

File: lightbulb-challenge-sammy/src/App.js (hunk 3 above)
Comment (Dgfarm02) [reply lightbulb-challenge-sammy/src/App.js:10]: I agree with James. Instructions are always good

File: lightbulb-challenge-sammy/src/components/LightSwitch.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    // State Values set at false = light switch 'Off'
+    const [lightSwitchStatus, setLightSwitchStatus] = useState(false)
+
+    // Function that updates State Values (sets it value to either true / false)
+    const lightsOnOff = () => {
+        setLightSwitchStatus(lightSwitchStatus === false ? true : false)  
+    }
+    
+    // if light switch is true = 'On', else 'Off'
+    let switchOnOff = lightSwitchStatus === true ? 'On' : 'Off'
```
Comment (Dgfarm02) [inline lightbulb-challenge-sammy/src/components/LightSwitch.js:15]: clean af

**Review by Dgfarm02 — COMMENTED** [review]
You have a lot of extra documents on yours which was a bit confusing but it looks like you've got the code down!

File: lightbulb-challenge-sammy/src/App.js (hunk 3 above)
Comment (MLMunoz85) [reply lightbulb-challenge-sammy/src/App.js:10]: I also like that you added instructions for the user.

**Pull request merged**